docs/application/tizen-studio/web-tools/config-editor.md

```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
 	</tbody>
 </table>
 
+<a name="ww_launch_screen"></a>
+### Tizen Launch Screen
+
+<table>
+	<thead>
+		<tr>
+			<th>&lt;tizen:launch_screen/&gt; element</th>
+		</tr>
+		</thead>
+		<tbody>
+		<tr>
+			<td>Used to indicate the splash screen.
+			<p><strong>Occurrences:</strong></p>
+			<ul>
+				<li>0 or 1</li>
+			</ul>
+			<p><strong>Expected children:</strong></p>
+			<ul>
+				<li><code>tizen:ls_default</code>
+				<p>Used to indicate the default of the splash screen.</p>
+				<p>Occurrences:</p>
+				<ul>
+					<li>1</li>
+				</ul>
+				<p>Attributes:</p>
+				<ul>
+					<li><code>background_image</code>: mandatory, string</li>
+					<li><code>background_color</code>: mandatory, string</li>
+					<li><code>image</code>: mandatory, string</li>
+					<li><code>image_border</code>: mandatory, string</li>
+				</ul>
+				</li>
+				<li><code>tizen:ls_landscape</code>
+				<p>Used to indicate the landscape of the splash screen.</p>
+				<p>Occurrences:</p>
+				<ul>
+					<li>0 or 1</li>
+				</ul>
+				<p>Attributes:</p>
+				<ul>
+					<li><code>background_image</code>: mandatory, string</li>
+					<li><code>background_color</code>: mandatory, string</li>
+					<li><code>image</code>: mandatory, string</li>
+					<li><code>image_border</code>: mandatory, string</li>
+				</ul>
+				</li>
+				<li><code>tizen:ls_portrait</code>
+				<p>Used to indicate the portrait of the splash screen.</p>
+				<p>Occurrences:</p>
+				<ul>
+					<li>0 or 1</li>
+				</ul>
+				<p>Attributes:</p>
+				<ul>
+					<li><code>background_image</code>: mandatory, string</li>
+					<li><code>background_color</code>: mandatory, string</li>
+					<li><code>image</code>: mandatory, string</li>
+					<li><code>image_border</code>: mandatory, string</li>
+				</ul>
+				</li>
+			</ul>
+			<p><strong>Attributes:</strong></p>
+			<ul>
+				<li><code>ready_when</code>
+				<p>Mandatory; string (available values: <code>first-paint</code> (default), <code>complete</code>, <code>custom</code>)</p>
+				<ul>
+					<li><code>first-paint</code>: hide splash screen when "frame,rendred" callback is called</li>
```

Comment:
Suggest: Please add "the" and use code tag:

hide splash screen when the `frame,rendred` callback is called

Also, please verify the spelling of rendered in the name of the callback.

Reply:
done